torchci/rockset/metrics/__sql/master_commit_red_avg.sql

```diff
@@ -26,33 +22,9 @@ WITH all_jobs AS (
         AND push.repository.name = 'pytorch'
         AND push._event_time >= PARSE_DATETIME_ISO8601(:startTime)
         AND push._event_time < PARSE_DATETIME_ISO8601(:stopTime)
-    UNION ALL
-    SELECT
-        push._event_time AS time,
-        CASE
-            WHEN job.job.status = 'failed' then 'failure'
-            WHEN job.job.status = 'canceled' then 'cancelled'
-            ELSE job.job.status
-        END AS conclusion,
-        push.head_commit.id AS sha,
-        IF(:useRetryConclusion,
-            ROW_NUMBER() OVER(PARTITION BY job.name, push.head_commit.id ORDER BY job.run_attempt DESC),
-            ''
-        ) AS attempt
-    FROM
-        circleci.job job
-        JOIN push ON job.pipeline.vcs.revision = push.head_commit.id
-    WHERE
-        push.ref IN ('refs/heads/master', 'refs/heads/main')
-        AND push.repository.owner.name = 'pytorch'
-        AND push.repository.name = 'pytorch'
-        AND push._event_time >= PARSE_DATETIME_ISO8601(:startTime)
-        AND push._event_time < PARSE_DATETIME_ISO8601(:stopTime)
```

Comment:
repos from other domains still use CircleCI, right? Lets leave this in for now in case other repos will use it

Reply:
Hmm, I see.  I try to remove Circle CI part here as an attempt to make this query more lightweight.  But if it's still needed, I will add it back